tp_timesheet/__main__.py

```diff
@@ -76,13 +70,25 @@ def parse_args():
     parser.add_argument(
         "-t",
         "--task",
-        type=str,
-        default="live",
-        choices=["holiday", "live", "OOO", "training"],
+        action="append",
+        nargs=2,
+        metavar=("task", "hour"),
         help="The type of task for the clockify submission. Specify task name "
-        + "if it is anything other than 'live'. Put the task name (e.g., training, OOO, holiday)",
+        + "if it is anything other than 'live'. Put the task name(idle, training, issue, OOO, holiday).\n"
+        + 'Passing multiple task-hour pair is also acceptable. (ex, "--task live 4 --task OOO 4" for afternoon OOO.)',
```

Comment:
```suggestion
        + 'Passing multiple task-hour pair is also acceptable. (E.g. "--task live 4 --task OOO 4" for afternoon OOO.)',
```
`e.g.` means "for example"